### ISSUE TITLE
Create helper function for adding names of countries for different types of  area codes 

### ISSUE THREAD

**Issue by malexan** [SWS-Methodology/faoswsTrade#86]
It will be useful for human-friendly reports.

- UN M49 (+UNSD version)
- Eurostat Geonomenclature
- FAO area list

**Comment by malexan** [comment]
I will use functions' names like that:
* area_name_fao
* area_name_m49 (including UNSD updates)
* area_name_geonom

Additionally we can create `suggest_area_type`: it will check given subset of area codes and make assumption on original classification.

To avoid manual creation of columns with area names `rprt_fulltable` and `rprt_writetable` can automatically add columns with area names based on existing columns with reporter and partner column names.

@muuankarski, do you have already anything for this?

**Comment by muuankarski** [comment]
Would be very useful.   

FAOSTAT-package has functions 

1. [`translateCountryCode`](https://github.com/mkao006/FAOSTATpackage/blob/master/Codes/R/translateCountryCode.R), and
2. [`fillCountryCode()`](https://github.com/mkao006/FAOSTATpackage/blob/master/Codes/R/fillCountryCode.R) 

that looks match from this table: https://github.com/mkao006/FAOSTATpackage/blob/master/FAOcountryProfile.csv - which could be already out of date?

Pocketbook uses them along with [`countrycode::countrycode`](https://github.com/vincentarelbundock/countrycode/blob/master/R/countrycode.R) and for Russian translations my variant [`countrycode.multilang::countrycode()`](https://github.com/muuankarski/countrycode.multilang/blob/master/R/countrycode.R)

**Comment by malexan** [comment]
More or less solved. I decided to postpone with guessing of codes class, as such function would be suitable for interactive use only, but is not safe for sws-module internals.

Currently all data sets are taken from the package itself. Later it should be changed to SWS source (see #24) to allow SWS users to update these data sets.

I dropped the idea of separate functions for each codes classification because of `dplyr::mutate_at` behavior. I tried to construct name of applied function as string with `paste0`. But when you pass string as a function name, `mutate_at` fails to find function. So I switched to `name_area()` with `code_class` parameter. 

Currently `name_area()` adds columns with area names to the end of data set. I would like to change it to have column with area name immediately after column with area code.
